src/project/view/templatepaintview.h

```diff
@@ -49,6 +49,8 @@ private slots:
 private:
     void onNotationSetup() override;
 
+    QString sequenceByActionCode(const actions::ActionCode& code) const;
```

Comment:
May be better rename to something like `shortcutsTitleByActionCode`

Reply:
renamed